### PR TITLE
Add conditional code to vde_cryptcab so it can compile under MacOS.

### PR DESCRIPTION
**Pull request by jguillaumes**
Please notice there is no wolfssl package for homebrew, so libwolfssl must be built by hand.

### CODE REVIEW

File: src/vde_cryptcab/vde_cryptcab_client.c [hunk 1]
```diff
@@ -96,9 +99,17 @@ rcv_challenge(struct datagram *pkt, struct peer *p)
...
 
 	close (fd);
-
-	memset(keyname + strlen(keyname) - 10, 'X', 6);
-	od = mkostemps(keyname, 4, O_RDWR | O_CREAT | O_TRUNC);
+	  memset(keyname + strlen(keyname) - 10, 'X', 6);
+#ifdef VDE_DARWIN
+	od = mkostemps(keyname, 4, O_EXLOCK);
+#else
+#ifdef VDE_DARWIN
```
Comment (danielinux) [inline src/vde_cryptcab/vde_cryptcab_client.c:106]: This check is repeated twice

Comment (danielinux) [reply src/vde_cryptcab/vde_cryptcab_client.c:106]: Please rebase on latest master and force-push. Remove the duplicate check.

**Review by danielinux — CHANGES_REQUESTED** [review]
- remove duplicate include "config.h"
- remove duplicate check for VDE_DARWIN 
- rebase on latest master

File: src/vde_cryptcab/vde_cryptcab_client.c [hunk 2]
```diff
@@ -8,7 +8,10 @@
  *
  */
 
+#include "config.h"
```
Comment (danielinux) [inline src/vde_cryptcab/vde_cryptcab_client.c:11]: Duplicated include statement

**Comment by jguillaumes** [comment]
This is embarrassing...  :( I probably hosed it when I tried to merge the upstream changes. I'm sorry... :/ 
To add a little bit more embarrasement I decided to try the pull request first in MY own fork and sent it to this project instead. So there is another closed pull request ...

**Pull request closed without merging**